src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -15,4 +15,8 @@
   <ItemGroup>
     <PackageReference Include="System.Diagnostics.DiagnosticSource" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="Internal\Shims\*.*" />
```

Comment:
Would it make more sense to put these in https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/Shared instead? I think it is a bit confusing to put them in this folder, but not actually used by this project.